lib/src/clojure_lsp/queries.clj

```diff
@@ -1,5 +1,8 @@
 (ns clojure-lsp.queries
```

Comment:
Lots of new ifs here, but expected.
After merge, let's try to fix all missing points to let dep-graph be the default soon, any fix on queries between that will probably be harder since we will need to keep testing both, glad we have the tests running for both states.

Reply:
Yep, lots of new ifs, but I think they're all contained to this namespace. And I think most new scenarios will be able to use the existing helpers, so there won't be too many additional test cases. But still, I agree, I don't want to live in this limbo world for too long.

Reply:
👍 on the tests... thanks for figuring those out. I'm glad we have them running for both states too.